### ISSUE TITLE
性别预测不是二分类吗？激活函数为何用softmax而不是sigmoid

### ISSUE THREAD

**Issue by CoderBinGe** [wujiekd/2020-Tencent-advertising-algorithm-contest-rank11#1]
`pred = Dense(2, activation='softmax')(x)
model.compile(loss='categorical_crossentropy',optimizer=AdamW(lr=3e-4, weight_decay=0.08, ), metrics=["accuracy"])
`
